adapters/socketio_berkeley.c

```diff
@@ -650,7 +652,11 @@ int socketio_setoption(CONCRETE_IO_HANDLE socket_io, const char* optionName, con
         }
         else if (strcmp(optionName, "tcp_keepalive_time") == 0)
         {
+#ifdef __APPLE__
```

Comment:
[https://github.com/Azure/azure-c-shared-utility/pull/16](https://github.com/Azure/azure-c-shared-utility/pull/16)

---
In reply to: [75419320](https://github.com/Azure/azure-c-shared-utility/pull/21#discussion_r75419320) [](ancestors = 75419320)